tasks/app/views/tasks/new.html.erb

```diff
@@ -2,7 +2,8 @@
   <h1 id="page-title" class="title">Enter Details</h1>
 </header>
 
-<%= form_for @task do |f| %>
+<%= render partial: 'form', locals: {method_type: :patch, button_text: "Add"} %>
+<!-- <%= form_for @task do |f| %>
```

Comment:
Even though this is inside of an HTML comment, the ERB tags will actually still be evaluated. So, this page will include the form from the partial, and then also have a second form in the final HTML output, but that second form will be inside of a comment.

This shouldn't be much of an issue in this case, but that code could eventually break in the future even though it's not being used at all. Since we have Git and can always access old versions of our code, it's preferable here to simply delete the old form code entirely.